apps/src/studio/studio.js

```diff
@@ -554,6 +554,17 @@ function callHandler (name, allowQueueExtension) {
   });
 }
 
+/**
+ * Calls provided function using our api - i.e. treats func as if it's code
+ * generated from blocks.
+ */
+Studio.callApiCode = function (name, func) {
```

Comment:
Having a tough time grokking this. Does this append a function to the event handler command queue with that given name?